qiskit/circuit/quantumcircuit.py

```diff
@@ -1533,6 +1534,380 @@ def _rebind_definition(self, instruction, parameter, value):
+        from .library.standard_gates.h import HGate
+        return self.append(HGate(), [qubit], [])
+
+    @deprecate_arguments({'ctl': 'control_qubit', 'tgt': 'target_qubit'})
+    def ch(self, control_qubit, target_qubit,  # pylint: disable=invalid-name
+           *, label=None, ctrl_state=None, ctl=None, tgt=None):  # pylint: disable=unused-argument
+        """Apply :class:`~qiskit.circuit.gates.CHGate`."""
+        from .library.standard_gates.h import CHGate
+        return self.append(CHGate(label=label, ctrl_state=ctrl_state),
+                           [control_qubit, target_qubit], [])
+
+    @deprecate_arguments({'q': 'qubit'})
+    def i(self, qubit, *, q=None):  # pylint: disable=unused-argument
+        """Apply :class:`~qiskit.circuit.gates.IGate`."""
+        from .library.standard_gates.i import IGate
+        return self.append(IGate(), [qubit], [])
+
+    @deprecate_arguments({'q': 'qubit'})
+    def id(self, qubit, *, q=None):  # pylint: disable=invalid-name,unused-argument
+        """Apply :class:`~qiskit.circuit.gates.IGate`."""
+        return self.i(qubit)
+
+    @deprecate_arguments({'q': 'qubit'})
+    def iden(self, qubit, *, q=None):  # pylint: disable=unused-argument
+        """Deprecated identity gate."""
+        warnings.warn('The QuantumCircuit.iden() method is deprecated as of 0.14.0, and '
+                      'will be removed no earlier than 3 months after that release date. '
+                      'You should use the QuantumCircuit.i() method instead.',
+                      DeprecationWarning, stacklevel=2)
+        return self.i(qubit)
+
+    def ms(self, theta, qubits):  # pylint: disable=invalid-name
+        """Apply :class:`~qiskit.circuit.gates.MSGate`."""
+        from .library.standard_gates.ms import MSGate
+        return self.append(MSGate(len(qubits), theta), qubits)
+
+    @deprecate_arguments({'q': 'qubit'})
+    def r(self, theta, phi, qubit, *, q=None):  # pylint: disable=invalid-name,unused-argument
+        """Apply :class:`~qiskit.circuit.gates.RGate`."""
+        from .library.standard_gates.r import RGate
+        return self.append(RGate(theta, phi), [qubit], [])
+
+    def rccx(self, control_qubit1, control_qubit2, target_qubit):
+        """Apply :class:`~qiskit.circuit.gates.RCCXGate`."""
+        from .library.standard_gates.x import RCCXGate
+        return self.append(RCCXGate(), [control_qubit1, control_qubit2, target_qubit], [])
+
+    def rcccx(self, control_qubit1, control_qubit2, control_qubit3, target_qubit):
+        """Apply :class:`~qiskit.circuit.gates.RC3XGate`."""
+        from .library.standard_gates.x import RC3XGate
+        return self.append(RC3XGate(),
+                           [control_qubit1, control_qubit2, control_qubit3, target_qubit],
+                           [])
+
+    @deprecate_arguments({'q': 'qubit'})
+    def rx(self, theta, qubit, *, label=None, q=None):  # pylint: disable=invalid-name,unused-argument
```

Comment:
```suggestion
    # pylint: disable=invalid-name,unused-argument
    def rx(self, theta, qubit, *, label=None, q=None): 
```
throws line-too-long error